### PR TITLE
Add info on golangci-lint

### PR DESCRIPTION
**Pull request by fiunchinho**
Adding help on how to use golangci-lint from intellij.

### CODE REVIEW

**Comment by fiunchinho** [comment]
Good idea! Please take a look.

**Comment by stone-z** [comment]
I would also do it in reverse -- e.g. add something about "If you use intellij, you can find tool info here" to the other page as well, so that people can find this

File: go/general_style_guide.md [hunk 1]
```diff
@@ -39,3 +39,8 @@ This should be used sparingly to avoid forming a bad habit of simply silencing b
 
 Details for using all of these methods can be found in the
 [False Positives](https://github.com/golangci/golangci-lint#false-positives) section of the docs.
+
+## Tooling
```
Comment (fiunchinho) [inline go/general_style_guide.md:43]: Not sure if it's the best header.

**Comment by fiunchinho** [comment]
Done

**Pull request merged**